src/WCSimPhysicsList.cc

```diff
@@ -223,16 +224,16 @@ void WCSimPhysicsList::ConstructEM()
 void WCSimPhysicsList::ConstructlArStepLimiter(){
 
 #ifdef GEANT4_7_0
-  theParticleIterator->reset();
-  while( (*theParticleIterator)() ){
+  aParticleIterator->reset();
```

Comment:
I don't think this matters too much, but I don't have a copy of 4.7.0 to test against.  I think if the pointer is declared to be aParticleIterator rather than theParticleIterator in the code, then it shouldn't matter.  If not, this can be reverted since, as you already mentioned, it's for a specific (and old!) version of GEANT4.